src/sentry/models/apiauthorization.py

```diff
@@ -36,6 +37,18 @@ class Meta:
         app_label = "sentry"
         db_table = "sentry_apiauthorization"
         unique_together = (("user", "application"),)
```

Comment:
@wedamija I kept this other index for now and will remove it in another PR.

Reply:
Ok, either way is fine - if you wanted to put it in this pr it'd be ok, you'd just want to make sure it runs after adding the new ones